ls1intum/Artemis
# `Exam mode`: Fix missing clone repository button for test runs

### Checklist
#### General
#### Client

### Motivation and Context
When running an exam test run, no clone repository button was displayed for programming exercises.

### Description
This issue was introduced by #6173 which aimed to hide the practice participation if it's possible to start a normal participation. However, the test run attribute is also used in the exam context. Here, starting the exercise is not possible, and the clone repository button should be shown for the test run.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Exam with Programming Exercises

1. Start a Test Run for the exam
2. Check that the clone repository button is visible
3. Also ensue that the issue described in #6173 is stil fixed.

### Review Progress

#### Code Review
#### Manual Tests

### Test Coverage


### Screenshots
